pkg/csi/service/common/vsphereutil.go

```diff
@@ -566,7 +566,7 @@ func isExpansionRequired(ctx context.Context, volumeID string, requestedSize int
 		},
 	}
 	// Query only the backing object details.
-	queryResult, err := utils.QueryVolumeUtil(ctx, manager.VolumeManager, queryFilter, querySelection, useAsyncQueryVolume)
```

Comment:
Change looks good but is there any caller to `QueryVolumeUtil` now? If not we can remove that too?

Reply:
Yes, we were calling QueryVolume() at 2 places in syncer and I am leaving it as it was before. So `QueryVoumeUtil` will be used by these 2 callers
![Screen Shot 2021-05-24 at 8 30 46 AM](https://user-images.githubusercontent.com/14131348/119370795-8c387e80-bc6a-11eb-8bd0-d300ec025d60.png)